content/en/docs/tasks/debug/debug-application/determine-reason-pod-failure.md

```diff
@@ -16,66 +16,68 @@ put in a termination message should also be written to
 the general
 [Kubernetes logs](/docs/concepts/cluster-administration/logging/).
 
-
-
-
 ## {{% heading "prerequisites" %}}
 
-
 {{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
```

Comment:
If you're cleaning this up, omit `{{< version-check >}}`. All supported Kubernetes versions let you read termination messages.

Reply:
Thank you, I really need to pay special attention to this detail.